### PR TITLE
Swarm/k8s : Catalogue Software update 4.0.0 and some fixes, upgrades

### PR DESCRIPTION
**Pull request by pranavv0**
Update:
1. added Rating and Databroker Verticles.
2.  updated images tag
3.  respective update in Examples values
4. Updated configs

### CODE REVIEW

**Comment by abhi4578** [comment]
@pranavv0 please also include necessary changes for swarm based deployment for catalogue

**Review by abhi4578 — CHANGES_REQUESTED** [review]
@pranavv0 please also include necessary changes for swarm based deployment for catalogue

**Comment by abhi4578** [comment]
Update readme with new values @pranavv0

**Comment by abhi4578** [comment]
Integrate profanity check deployment to  cat hem repo

File: K8s-deployment/Charts/catalogue/templates/profanity-checker-sdk/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+          volumeMounts:
+            - name: foo
+              mountPath: bar
+        {{- end }}
+        {{- if .Values.profanityCheckSdk.initContainers }}
+          {{- include "common.tplvalues.render" (dict "value" .Values.profanityCheckSdk.initContainers "context" $) | nindent 8 }}
+        {{- end }}
+      containers:
+        - name: profanity-check-sdk
+          image: {{ template "catalogue.image" . }}
```
Comment (abhi4578) [inline K8s-deployment/Charts/catalogue/templates/profanity-checker-sdk/deployment.yaml:69]: How is profanity check sdk image set?

File: K8s-deployment/Charts/catalogue/templates/profanity-checker-sdk/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+          {{- end }}
+          volumeMounts:
+          {{- if .Values.profanityCheckSdk.extraVolumeMounts }}
+          {{- include "common.tplvalues.render" (dict "value" .Values.profanityCheckSdk.extraVolumeMounts "context" $) | nindent 12 }}
+          {{- end }}
+        {{- if .Values.profanityCheckSdk.sidecars }}
+        {{- include "common.tplvalues.render" ( dict "value" .Values.profanityCheckSdk.sidecars "context" $) | nindent 8 }}
+        {{- end }}
+          ports:
+            - name: hazelcast
```
Comment (abhi4578) [inline K8s-deployment/Charts/catalogue/templates/profanity-checker-sdk/deployment.yaml:97]: Not needed. hazelcast port

File: K8s-deployment/Charts/catalogue/templates/profanity-checker-sdk/deployment.yaml (hunk 1 above)
Comment (pranavv0) [reply K8s-deployment/Charts/catalogue/templates/profanity-checker-sdk/deployment.yaml:69]: Yet to commit that

File: K8s-deployment/Charts/catalogue/templates/profanity-checker-sdk/deployment.yaml (hunk 2 above)
Comment (pranavv0) [reply K8s-deployment/Charts/catalogue/templates/profanity-checker-sdk/deployment.yaml:97]: okk

**Review by abhi4578 — CHANGES_REQUESTED** [review]
LGTM until now. Add 
1. please also include necessary changes for swarm based deployment for catalogue
2. Add values to Readme file

**Review by abhi4578 — APPROVED** [review]
LGTM

**Pull request merged**